test/integration/builder/additional.js

```diff
@@ -758,7 +758,7 @@ module.exports = function(knex) {
         expect(addTimeout).to.throw(
           'Query cancelling not supported for this dialect'
         );
-        return;
+        return; // TODO: Use `this.skip()` here?
```

Comment:
In this case, it looks like there should potentially be 2 separate test cases.  The 2nd test case is only applicable to a subset of Dialects.